src/applications/mhv-medical-records/util/constants.js

```diff
@@ -172,6 +173,30 @@ export const vitalTypes = {
   HEIGHT: ['HEIGHT', 'BODY_HEIGHT'],
 };
 
+// Grouped LOINC codes for each canonical vital type. This is the single source of truth.
+// Any additions (new LOINC variants) only need to be appended here and will automatically
+// be reflected in loincToVitalType and allowedVitalLoincs.
+export const vitalLoincGroups = {
+  BLOOD_PRESSURE: [loincCodes.BLOOD_PRESSURE],
+  PULSE: [loincCodes.HEART_RATE],
+  RESPIRATION: [loincCodes.BREATHING_RATE],
+  PULSE_OXIMETRY: [loincCodes.PULSE_OXIMETRY_1, loincCodes.PULSE_OXIMETRY_2],
+  TEMPERATURE: [loincCodes.TEMPERATURE],
+  WEIGHT: [loincCodes.WEIGHT, '3141-9'], // include common alternate weight LOINC 3141-9
```

Comment:
was this an isolated case for just weight, or do we want to make this a little bit more robust so we're not hard coding LOINCs